README.md

```diff
@@ -103,25 +103,20 @@ After getting the dependencies install it with:
 
 ```bash
 git submodule update --init --recursive
-./autogen.sh
-# on macOS you need to tell linkers how to reach home-brewed sqlite3 and curl:
-# export LDFLAGS="-L/usr/local/opt/sqlite/lib/ -lsqlite3" PKG_CONFIG_PATH=/usr/local/opt/curl/lib/pkgconfig
-./configure
-make test -j$(nproc)
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
+make -j$(nproc)
 sudo make install
-# Note: on macOS, a few tests involving time formatting will fail: narrativebuilder, util_odin, mapmatch
 ```
 
-Please see `./configure --help` for more options on how to control the build process. There are a few notable options that you might want to try out:
+For more build options run the interactive GUI:
 
-* `--enable-data-tools=no` will disable building any of the components (library bits, executables and tests) which can be used to create the data that the services run on. This can be useful in embedded situations where you really don't need some of the dependencies above.
```

Comment:
I think it is better to keep the section and only change configure arguments to:
-DENABLE_DATA_TOOLS=OFF
 -DENABLE_PYTHON_BINDINGS=OFF
 -DENABLE_SERVICES=OFF
-DBUILD_SHARED_LIBS=Off